### ISSUE TITLE
Clarification on `sloika_hdf5s` contents of "training data" dataset

### ISSUE THREAD

**Issue by newalexander** [rrwick/Basecalling-comparison#23]
Hello, and thanks for making this work easily available!

I have a question about the contents of the "training data" dataset (https://bridges.monash.edu/articles/dataset/Training_data/7676198) from the Genome Biology paper (https://doi.org/10.1186/s13059-019-1727-y). 

Each entry in this dataset contains a `sloika_hdf5s` file with the following items: `bad`, `chunks`, `labels`, and `weights`. `chunks` is describer in the paper:

    Sloika subdivides these reads into ‘chunks’ of 4000 signal values, of which there were 7,693,885.

but the other items are less obvious in meaning. For example, `labels` takes on values between 0 and 1024, so it doesn't correspond to the sequence labels of the corresponding FASTA file.

Clarification here would be helpful, thanks!

**Comment by myrtlecat** [comment]
Hi there. I'm not directly involved with this project, but I work as a researcher at ONT.

`sloika` was deprecated around 5 years ago! But to the best of my knowledge, the `labels` values encode 5-mers found in the reference fasta padded with a "blank" label. Apologies I can't be more helpful, but `sloika` is an ancient relic by now.

If you are interested in training models to basecall nanopore signal I would strongly recommend starting fresh with [`bonito`](https://github.com/nanoporetech/bonito).
